pkg/kubernetes/kube_test.go

```diff
@@ -364,7 +400,36 @@ func resourceFromYaml(resourceFileName string) (*unstructured.Unstructured, erro
 
 	return resource, nil
 }
+func multipleResourcesFromYaml(resourceFileName string) ([]KubernetesResource, error) {
```

Comment:
You dont need to duplicate these functions (`multipleResourcesFromYaml` `getResourceFromString`), do you? Just write Unit Tests for the ones in the `util` pkg and reuse them here. Although the Unit Test you added above should be using `MultiResourceOperation` which already calls theses.

Reply:
done

Reply:
I still see `multipleResourcesFromYaml` here. I would recommend hardcoding the resources you expect to get from the function being tested (`MultiResourceOperation`) like so:

https://github.com/keikoproj/kubedog/blob/0df36b2b55a7b0858860cafdef8ad0d4b11a98ea/pkg/common/common_test.go#L238-L243

Using a function to be the judge of the test of another function sounds contradicting to me. You would need a test for the function judging since that one might have errors as well.

You have `yamlSeparator`, `trimTokens` and `KubernetesResource` duplicated, you have this things defined in the `util` `pkg`, use them from there.

Reply:
Just a suggestion: this test would be highly benefited from table driven tests like done in [`common_test.go`](https://github.com/keikoproj/kubedog/blob/0df36b2b55a7b0858860cafdef8ad0d4b11a98ea/pkg/common/common_test.go). The other tests in this pkg do not have it because I did not know of its existence back when I wrote them, but they need refactoring as well.

Reply:
I refactored the test to be more like the tests in common_test.go. The new test matches the function output's ApiVersion, Name, and Kind against hardcoded expected values.